src/transformers/blenderbot-90M-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "embedding_size": 512,
+  "n_layers": 8,
+  "ffn_size": 2048,
+  "dropout": 0.1,
+  "n_heads": 16,
+  "n_positions": 512,
+  "activation": "gelu",
+  "attention_dropout": 0.0,
+  "relu_dropout": 0.0,
+  "learn_positional_embeddings": true,
```

Comment:
I think here you either use `sinusoidal_position_embeddings` *or* you learn `positional_position_embeddings`. It's normally the default case to learn the positional embeddings, so I would use the parameter `use_sinusoidal_position_embeddings` here and set it to `False` :-)